src/style.css

```diff
@@ -0,0 +1,93 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf) format("TrueType");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+  width: 100%;
+  margin-top: 20px;
+  position: relative;
+  height: 70px;
+}
+
+.form-small {
+  height: 42px;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.search-bar {
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  height: 70px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-size: 16px;
+  padding-left: 62px;
+  background-color: #fff;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: bold;
+  text-shadow: 0 3px 3px rgb(190, 186, 186);
+}
+
+.search-bar-small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  font-weight: 400;
+  font-style: normal;
+  width: 100%;
+  line-height: 19px;
+}
+
+.search-bar-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  text-shadow: none;
+}
+
+.search-bar-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: bold;
+  text-shadow: none;
+}
```

Comment:
DRY. You have a lot of the same code for :hover and :focus in different classes. Move it under the common class.